tests/integration/test_endpoints.py

```diff
@@ -138,4 +137,4 @@ def test_deltas_post(monkeypatch, client):
         }],
     })
     assert response.status_code == 200
-    assert len(response.json['operations']) == 72
+    assert len(response.json['operations']) == 335
```

Comment:
How come this number increased so dramatically? Are we simply more verbose in logging operations? Is this a problem we need to address with regard to bloat in the JSON?

Reply:
This is due to how the medium is applied now. Previously, we'd [set the lower bound on all reactions in `model.medium` to 0](https://github.com/DD-DeCaF/model/commit/dcd736be6f32949ddd06d9203e4dd104f0e11d4a#diff-ee5d9931e30cd22333ac83c04ad23f8cL48). Now, we're simply [assigning `model.medium`](https://github.com/DD-DeCaF/model/commit/dcd736be6f32949ddd06d9203e4dd104f0e11d4a#diff-ee5d9931e30cd22333ac83c04ad23f8cR108), letting cobrapy decide which bounds need changing.

That means we need one operation to modify each exchange reaction, since we don't know which ones changed, and that results in a lot more operations (some of which likely haven't actually changed).